test/active_model_serializers/test/serializer_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'rails-controller-testing' if Rails::VERSION::MAJOR >= 5
```

Comment:
@bf4 I am ok with that, I think you can follow the same idea you did on this https://github.com/rails-api/active_model_serializers/pull/1322 squash all these commits into a single one, create a new PR and close this one, and @mention me so I can follow up :smile: